src/tools/js-wrapper.h

```diff
@@ -23,64 +23,68 @@ namespace wasm {
-         "console.log('[LoggingExternalInterface logging ' + literal(x, 'i32') "
-         "+ ']') },\n"
-         "    'log-i64': function(x, y) { "
-         "console.log('[LoggingExternalInterface logging ' + literal(x, 'i32') "
-         "+ ' ' + literal(y, 'i32') + ']') },\n" // legalization: two i32s
-         "    'log-f32': function(x)    { "
-         "console.log('[LoggingExternalInterface logging ' + literal(x, 'f64') "
-         "+ ']') },\n" // legalization: an f64
-         "    'log-f64': function(x)    { "
-         "console.log('[LoggingExternalInterface logging ' + literal(x, 'f64') "
-         "+ ']') },\n"
-         "  },\n"
-         "  'env': {\n"
-         "    'setTempRet0': function(x) { tempRet0 = x },\n"
-         "    'getTempRet0': function() { return tempRet0 },\n"
-         "  },\n"
-         "});\n";
+  ret +=
+    "if (typeof console === 'undefined') {\n"
+    "  console = { log: print };\n"
+    "}\n"
+    "var tempRet0;\n"
+    "var binary;\n"
+    "if (typeof process === 'object' && typeof require === 'function' /* "
+    "node.js detection */) {\n"
+    "  var args = process.argv.slice(2);\n"
+    "  binary = require('fs').readFileSync(args[0]);\n"
+    "  if (!binary.buffer) binary = new Uint8Array(binary);\n"
+    "} else {\n"
+    "  var args;\n"
+    "  if (typeof scriptArgs != 'undefined') {\n"
+    "    args = scriptArgs;\n"
+    "  } else if (typeof arguments != 'undefined') {\n"
+    "    args = arguments;\n"
+    "  }\n"
+    "  if (typeof readbuffer === 'function') {\n"
+    "    binary = new Uint8Array(readbuffer(args[0]));\n"
+    "  } else {\n"
+    "    binary = read(args[0], 'binary');\n"
+    "  }\n"
+    "}\n"
+    "function literal(x, type) {\n"
+    "  var ret = type + '.const ';\n"
+    "  switch (type) {\n"
+    "    case 'i32': ret += (x | 0); break;\n"
+    "    case 'f32':\n"
+    "    case 'f64': {\n"
+    "      if (x == 0 && (1 / x) < 0) ret += '-';\n"
+    "      ret += x;\n"
+    "      break;\n"
+    "    }\n"
+    "    default: throw 'what?';\n"
+    "  }\n"
+    "  return ret;\n"
+    "}\n"
+    "var instance = new WebAssembly.Instance(new "
+    "WebAssembly.Module(binary), {\n"
+    "  'fuzzing-support': {\n"
+    "    'log-i32': function(x)     { "
+    "console.log('[LoggingExternalInterface logging ' + literal(x, 'i32') "
+    "+ ']') },\n"
+    "    'log-i64': function(x, y)  { "
+    "console.log('[LoggingExternalInterface logging ' + literal(x, 'i32') "
+    "+ ' ' + literal(y, 'i32') + ']') },\n" // legalization: two i32s
+    "    'log-f32': function(x)     { "
+    "console.log('[LoggingExternalInterface logging ' + literal(x, 'f64') "
+    "+ ']') },\n" // legalization: an f64
+    "    'log-f64': function(x)     { "
+    "console.log('[LoggingExternalInterface logging ' + literal(x, 'f64') "
+    "+ ']') },\n"
+    "    'log-nullref': function(x) { "
+    "console.log('[LoggingExternalInterface logging ' + literal(x, 'nullref') "
+    "+ ']') },\n"
```

Comment:
`log-nullref` is the only thing added here, but clang-format cased the whole string to be reindented.